Collections/Collections.tex

```diff
@@ -964,6 +980,21 @@ \subsection{Collecting results (\lct{collect:})}
 (1 to: 5) collect: [:each | each * 2] --> #(2 4 6 8 10)
 \end{code}
 
+\paragraph{Shortcuts.}
+
+If you look up the implementation of \ct{collect:}, \ct{do:}, \autc in any collection class, you will find that they evaluate the argument block by sending the message \mthind{BlockClosure}{value:} to it.

```

Comment:
\mthind{BlockClosure}{value:} is the method, so it should just be "value:"

Reply:
What do you mean? In the PDF, this will just appear as `value:`. `\mthind` is for the correct index only.